doc/tests.rst

```diff
@@ -1,3 +1,5 @@
+.. _writing-tests:
+
 *************************
 Writing and running tests
```

Comment:
I haven't looked over this document yet, but when I do, I'll change these headings to have all words capitalized: 
Writing and Running Tests

Reply:
:+1:  I didn't change this header as part of this changeset, just added the reference target, but I think that will stay the same.